hr_recruitment_firstname/models/hr_applicant.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+#
+##############################################################################
+#
+#    Authors: Adrien Peiffer
+#    Copyright (c) 2015 Acsone SA/NV (http://www.acsone.eu)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, api
+
+
+class HrApplicant(models.Model):
+    _inherit = 'hr.applicant'
+
+    partner_name = fields.Char(string="Applicant's Lastname")
```

Comment:
I think we should stick consistent with the other modules, by adding lastname and firstname fields, not to change the name field, so i suggest to add partner_lastname, partner_firstname and compute the partner_name fields based on those 2...